packages/cli/commands/project/dev.js

```diff
@@ -61,9 +61,11 @@ const { getHubSpotWebsiteOrigin } = require('@hubspot/local-dev-lib/urls');
 const {
   logApiErrorInstance,
   ApiErrorContext,
-  isMissingScopeError,
   isSpecifiedError,
 } = require('../../lib/errorHandlers/apiErrors');
+const {
+  isMissingScopeError,
+} = require('@hubspot/local-dev-lib/errors/apiErrors');
```

Comment:
I migrated over `isMissingScopeError` to the LDL version since its for a sandboxes endpoint, but left `isSpecifiedError` as the local version. The usage of `isSpecifiedError` in this file is for `handleProjectUpload` which is still pulling `uploadProject` from the old `@hubspot/cli-lib` dependency where the http base is still request-promise-native. 

Same goes for `cli -> lib -> projects.js` in this repo since `fetchProject` is also pulled in from `@hubspot/cli-lib`. Usage is limited to just those two in projects files - I'll leave a comment in the code to reduce any confusion once that migration happens